src/views/ManageList.tsx

```diff
@@ -76,6 +85,44 @@ export function ManageList({ listPath }: Props) {
 		}
 	};
 
+	const handleInvite = async (
+		e: FormEvent<HTMLFormElement>,
+		listPath: string,
+	) => {
+		console.log("Button clicked! Inviting user!");
+		e.preventDefault();
+
+		const trimmedEmailName = validateTrimmedString(emailName);
+
+		if (!trimmedEmailName) {
```

Comment:
Similar to my note above, we can remove this check: it's not possible to hit it.